packages/lit-html/src/lib/lit-html.ts

```diff
@@ -888,6 +917,58 @@ export class EventPart extends AttributePart {
   }
 }
 
+export class SpreadPart {
+  __value: unknown;
+  __directive?: Directive;
+
+  constructor(public __element: Element) {}
+
+  __setValue(value: unknown): void {
+    const directive = (value as DirectiveResult)._$litDirective$;
+    if (directive !== undefined) {
```

Comment:
It's unfortunate that each part type has to deal with directives. Can this be centralized?

Reply:
Kevin has part of this in #1426